### PR TITLE
Abilities balance change

### PR DESCRIPTION
**Pull request by 1ran-wang**

### CODE REVIEW

**Comment by 1ran-wang** [comment]
I'm using old branch to do the balance, so the change log has conflicts. :(

File: src/game/scripts/npc/abilities/custom/holdout_gods_strength.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@
 		"AbilityDuration" "25.0"
 		"AbilityManaCost" "125"
 		"ReduxFlags"                                                           "cooldown"
+		"ReduxBans"                                                            "sven_gods_strength"
```
Comment (darklordabc) [inline src/game/scripts/npc/abilities/custom/holdout_gods_strength.txt:11]: Why is this a troll combo with gods strength?

**Comment by 1ran-wang** [comment]
Both will have bonus str and bonus atk base on basic atk,almost one hit
anyone after LV6 and redux version have CDR, this build with can easily
keep high dmg with high hp in whole game.Will be Unstopable pick with
autogun/split shot/any mobility.

Blade mail won't hurt as he has more hp.
Silver doesn't work because it's not passive.
No more permstun build so he can eventually one hit you.



2017年10月15日 下午3:37，"darklordabc" <notifications@github.com>写道：

**Comment by darklordabc** [comment]
@TechiesTheOne for future, you should make a new branch everytime you want to create a new feature, creating new branch only takes a second and we can have an unlimited amount.

**Comment by 1ran-wang** [comment]
OK i got it.

2017年10月20日 下午11:50，"darklordabc" <notifications@github.com>写道：

@TechiesTheOne <https://github.com/techiestheone> for future, you should
make a new branch everytime you want to create a new feature, creating new
branch only takes a second and we can have an unlimited amount.

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/darklordabc/Legends-of-Dota-Redux/pull/1270#issuecomment-338361527>,
or mute the thread

**Pull request merged**